src/pipecat/services/noisereduce.py

```diff
@@ -0,0 +1,36 @@
+
+import noisereduce as nr
```

Comment:
We should add a try/catch here and advise the user to do `pip install pipecat[noisereduce]`

Reply:
We do this in other services.